src/client/app/types/redux/graph.ts

```diff
@@ -54,6 +54,7 @@ export interface ThreeDState {
 	meterOrGroupID: MeterOrGroupID | undefined;
 	meterOrGroup: MeterOrGroup | undefined;
 	readingInterval: ReadingInterval;
+	numDays: number | undefined; // Number of days for 3D graphic span (undefined means use default: 1 year)
```

Comment:
I'm curious about using undefined vs setting the default value to 1 year in src/client/app/redux/slices/graphSlice.ts.